[2020-01-14 09:09:28,615] {taskinstance.py:630} INFO - Dependencies all met for <TaskInstance: initial_build.populate_tables 2020-01-14T09:09:09.210412+00:00 [queued]>
[2020-01-14 09:09:28,627] {taskinstance.py:630} INFO - Dependencies all met for <TaskInstance: initial_build.populate_tables 2020-01-14T09:09:09.210412+00:00 [queued]>
[2020-01-14 09:09:28,627] {taskinstance.py:841} INFO - 
--------------------------------------------------------------------------------
[2020-01-14 09:09:28,627] {taskinstance.py:842} INFO - Starting attempt 1 of 1
[2020-01-14 09:09:28,627] {taskinstance.py:843} INFO - 
--------------------------------------------------------------------------------
[2020-01-14 09:09:28,644] {taskinstance.py:862} INFO - Executing <Task(PythonOperator): populate_tables> on 2020-01-14T09:09:09.210412+00:00
[2020-01-14 09:09:28,644] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'initial_build', 'populate_tables', '2020-01-14T09:09:09.210412+00:00', '--job_id', '705', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/initial_build.py', '--cfg_path', '/tmp/tmpodlt4h6q']
[2020-01-14 09:09:30,015] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables [2020-01-14 09:09:30,014] {__init__.py:51} INFO - Using executor SequentialExecutor
[2020-01-14 09:09:30,015] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables [2020-01-14 09:09:30,015] {dagbag.py:92} INFO - Filling up the DagBag from /home/ubuntu/venv/airflow/dags/initial_build.py
[2020-01-14 09:09:30,019] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables [2020-01-14 09:09:30,018] {baseoperator.py:354} WARNING - schedule_interval is used for <Task(PythonOperator): create_tables>, though it has been deprecated as a task parameter, you need to specify it as a DAG parameter instead
[2020-01-14 09:09:30,019] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables [2020-01-14 09:09:30,019] {baseoperator.py:354} WARNING - schedule_interval is used for <Task(PythonOperator): populate_tables>, though it has been deprecated as a task parameter, you need to specify it as a DAG parameter instead
[2020-01-14 09:09:30,030] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables [2020-01-14 09:09:30,030] {cli.py:545} INFO - Running <TaskInstance: initial_build.populate_tables 2020-01-14T09:09:09.210412+00:00 [running]> on host ip-172-31-19-44.us-west-2.compute.internal
[2020-01-14 09:09:30,037] {python_operator.py:105} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=initial_build
AIRFLOW_CTX_TASK_ID=populate_tables
AIRFLOW_CTX_EXECUTION_DATE=2020-01-14T09:09:09.210412+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-01-14T09:09:09.210412+00:00
[2020-01-14 09:09:30,164] {taskinstance.py:1058} ERROR - table football_teams has no column named fb_id
Traceback (most recent call last):
  File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 930, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/ubuntu/venv/python_scripts/functions/populate_initial_tables.py", line 46, in populate_initial_tables
    (item['id'], item['name'], item['short_name']))
sqlite3.OperationalError: table football_teams has no column named fb_id
[2020-01-14 09:09:30,165] {taskinstance.py:1089} INFO - Marking task as FAILED.
[2020-01-14 09:09:30,187] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables Traceback (most recent call last):
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/bin/airflow", line 37, in <module>
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     args.func(args)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     return f(*args, **kwargs)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/bin/cli.py", line 551, in run
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     _run(args, dag, ti)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/bin/cli.py", line 469, in _run
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     pool=args.pool,
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     return func(*args, **kwargs)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 930, in _run_raw_task
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     result = task_copy.execute(context=context)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 113, in execute
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     return_value = self.execute_callable()
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     return self.python_callable(*self.op_args, **self.op_kwargs)
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables   File "/home/ubuntu/venv/python_scripts/functions/populate_initial_tables.py", line 46, in populate_initial_tables
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables     (item['id'], item['name'], item['short_name']))
[2020-01-14 09:09:30,188] {base_task_runner.py:115} INFO - Job 705: Subtask populate_tables sqlite3.OperationalError: table football_teams has no column named fb_id
[2020-01-14 09:09:33,603] {logging_mixin.py:112} INFO - [2020-01-14 09:09:33,603] {local_task_job.py:124} WARNING - Time since last heartbeat(0.00 s) < heartrate(5.0 s), sleeping for 4.996004 s
[2020-01-14 09:09:38,605] {logging_mixin.py:112} INFO - [2020-01-14 09:09:38,604] {local_task_job.py:103} INFO - Task exited with return code 1
